packages/test-helpers/package.js

```diff
@@ -4,7 +4,7 @@ Package.describe({
 });
 
 Package.onUse(function (api) {
-  api.use(['underscore', 'tracker', 'ejson', 'tinytest', 'random']);
+  api.use(['underscore', 'tracker', 'ejson', 'tinytest', 'random', 'blaze']);
```

Comment:
Only the `renderToDiv` helper depends on Blaze, so in theory we could let you use the other helpers without pulling in Blaze.  I guess pulling in Blaze doesn't do any harm, though.